docs/kubernetes/user-guides/snapshots.md

```diff
@@ -19,33 +21,102 @@ $ ./deploy/kubernetes/deploy-driver.sh
 
 ### Snapshot Example
 
-1. Create example Default Snapshot Class
+**Create Storage Class:**
+
+If you haven't created a `StorageClass` yet, create one first:
+
+```console
+kubectl apply -f ./examples/kubernetes/snapshot/storageclass.yaml
```

Comment:
can we just link to the `examples/kubernetes/demo-zonal-sc.yaml` file instead?

Reply:
Used `demo-zonal-sc.yaml` now.